core/src/test/java/org/testcontainers/junit/FixedHostPortContainerTest.java

```diff
@@ -14,28 +23,82 @@
  */
 public class FixedHostPortContainerTest {
 
-    private static final int REDIS_PORT = 6379;
+    /**
+     * Docker image for simple open-source HTTP echo server https://github.com/hashicorp/http-echo
+     */
+    private static final String HTTP_ECHO_IMAGE = "hashicorp/http-echo";
```

Comment:
There are a couple of http-based tests in `GenericContainer`'s tests, could you please reuse the approach?
Although this image is small, it would be better to not add more images to the test suite because it increases the traffic and execution time.
Thanks!

Reply:
You mean `.withCommand("/bin/sh", "-c", "while true; do echo \"$MAGIC_NUMBER\" | nc -l -p 80; done");`? Sure, will do. Copy-paste as hell